app/HoistAppModel.js

```diff
@@ -41,6 +41,13 @@ export function HoistAppModel(C) {
             get() {return false}
         },
 
+        /**
+         * Override in apps appModel to check if user has required role.
```

Comment:
Let's change the signature of this so that it actually gets passed the user object as the first argument.  Will just make things much simpler and self documenting as to what an application should typically do with this method.

message could be  'Access control not yet implemented for this application.  See AppModel.checkAccess()'

Also, can we update the comment?